features/main_screen/CurrencyPair/CurrencyPair.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { View, Text} from "react-native";
+import { styles } from "./style";
+import {ICurrencyPair} from "../PairsList";
+
+interface IProps {
+  pair: ICurrencyPair
+}
+
+const CurrencyPair = (props: IProps) => {
+  const {
+    pair
+  } = props;
+  const {
+    title,
+    ratio
+  } = pair;
+
+  return (
+    <View style={styles.sectionPair}>
```

Comment:
стили лучше сразу писать в виде массива, чтобы потом не дописовать, когда надо будет делать правки (сделай лайв темплейт себе на `style={[
    styles.sectionPair
    ]}` :
```suggestion
    <View
     style={[
    styles.sectionPair
    ]}>
```